### DISCUSSION TITLE
2024/03/beautiful-net-test-reports-using-github-actions/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#893] [Specific Post Comments]
# 2024/03/beautiful-net-test-reports-using-github-actions/

I know some folks who would like to move to GitHub Actions from Azure DevOps, but would really miss the “Test View” that Azure DevOps provides. Let’s see if we can work some magic with GitHub Actions to get us close to that within GitHub.

https://seankilleen.com/2024/03/beautiful-net-test-reports-using-github-actions/

**Comment by SeanKilleen** [comment]
Hey all -- Just wanted to say that for a little while, the snippets in this post were wrong. The variables in the GitHub Actions snippets got cut off because my blog's syntax parser thought they were variables within the blog post itself. This will be fixed shortly upon the deployment that's about to happen.

**Comment by tomaszek92** [comment]
Nice work, it helps me a lot :)

**Comment by andreaswurtz** [comment]
Nice work, very helpful. Had trouble running the windows version of the "EnricoMi/publish-unit-test-result-action" workflow, so ended up with saving the tests results from my windows workflow, and then running the publish test result action workflow in a linux workflow with test files from the finished windows workflow.
